webrtc.html

```diff
@@ -228,7 +228,8 @@ <h2>Dictionary {{RTCConfiguration}} Members</h2>
                 {{RTCPeerConnection}} implementation selects which of
                 the certificates is used for a given connection; how
                 certificates are selected is outside the scope of this
-                specification.</p>
+                specification. Existing implementations only utilize the
+                first certificate provided; the others are ignored.</p>
```

Comment:
Wrap <p class="note"> around the new sentence?

Reply:
Done.